### DISCUSSION TITLE
MediaElement Concerns

### DISCUSSION THREAD

**Discussion by mtyeager** [CommunityToolkit/Maui#323] [New Feature Discussions]
Running MediaElement on Android in a CollectionView - in a news feed. A news feed a standard, important and common scenario for internal, b2b and B2C users. 

AutoPlay set to False, plays anyway. It does start and stop as you scroll a video into view. But users hate it when video starts playing uninitiated. 

Also, I do not see a full screen option. At least on Android. Not sure about iOS.

Playback Controls appear to be Absolute Positioned. When you scroll the collectionview, the MediaElement scrolls fine, but the Playback Controls stay where they first appeared, as the news feed scrolls in the background. Is there currently a way to style and position the Playback Controls? 

From a news feed perspective, users want to upload their videos for others to enjoy. If the best way to do this is using Azure Media Services, then an end to end sample, from upload to locator url would be helpful. This would include ability to delete. It looks like MediaElement does not support streaming and therefore does not work with Azure Media Services. 

One would hope the MediaElement could work more like an Image. You just put it in the page, give it a url such as a streaming locator, and it just works. I realize this may be an extreme cross platform challenge, but video is so important these days, how can Xamarin/Maui prosper without it?

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
